mindboggle/mindboggle123

```diff
@@ -329,7 +329,7 @@ else:
         transformer_nn = Node(ApplyTransforms(), name="transformer_nn")
         transformer_nn.inputs.reference_image = IMAGE
         transformer_nn.inputs.dimension = 3
-        transformer_nn.inputs.invert_transform_flags = [False, False]
+        transformer_nn.inputs.invert_transform_flags = [False] * N
```

Comment:
this should not be the case. corticalthickness generates two transform files that go through the merge node. this change should be reverted.

Reply:
`N` on the other hand is the total number of atlases, which has no bearing on the quick option.

Reply:
this change should be in the fusion section, and invert_transform should be a single element vector with `False`